plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityCharged.java

```diff
@@ -1,135 +1,71 @@
 package com.denizenscript.denizen.objects.properties.entity;
 
 import com.denizenscript.denizen.objects.EntityTag;
-import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.Mechanism;
 import com.denizenscript.denizencore.objects.core.ElementTag;
-import com.denizenscript.denizencore.objects.properties.Property;
-import com.denizenscript.denizencore.objects.properties.PropertyParser;
-import org.bukkit.entity.Entity;
-import org.bukkit.entity.Guardian;
-import org.bukkit.entity.Vex;
-import org.bukkit.entity.WitherSkull;
-
-public class EntityCharged implements Property {
-
-    public static boolean describes(ObjectTag object) {
-        if (!(object instanceof EntityTag)) {
-            return false;
-        }
-        Entity entity = ((EntityTag) object).getBukkitEntity();
-        return entity instanceof WitherSkull
-                || entity instanceof Vex
-                || entity instanceof Guardian;
+import org.bukkit.entity.*;
+
+public class EntityCharged extends EntityProperty<ElementTag> {
+
+    // <--[property]
+    // @object EntityTag
+    // @name charged
+    // @input ElementTag(Boolean)
+    // @description
+    // If the entity is a wither skull, controls whether the skull is charged. Charged skulls are blue.
+    // If the entity is a vex, controls whether the vex is charging. Charging vexes have red lines.
+    // If the entity is a guardian, controls whether the guardian's laser is active.
+    // If the entity is a ghast, controls whether the ghast is charging. Charging ghasts have red eyes and a red mouth.

```

Comment:
Should ideally stick to the usual property meta format, maybe something like:
```
Controls whether an entity is charged:
- For a wither skull, this is whether it's a charged blue skull.
- For a vex, this is whether it's charging and has red lines.
- For a guardian, this is whether it's laser is active.
- For a ghast, this is whether it's charging and has red eyes and mouth.
```
(Do note that this is just an example I wrote up here, feel free to discuss this further on Discord).